From: Didier Raboud <odyx@debian.org>
Date: Tue, 9 Aug 2016 18:33:56 +0200
Subject: Tests: ignore errors triggered on ipv6-deprived hosts

Ignore the 'Address family not supported by protocol' error that breaks the
errorlines counting on buildds without a working IPv6 stack.
---
 test/run-stp-tests.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/run-stp-tests.sh b/test/run-stp-tests.sh
index a601581..0d4e88e 100755
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -1025,7 +1025,9 @@ else
 fi
 
 # Error log messages
-count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | \
+       $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | \
+       wc -l | awk '{print $1}'`
 if test $count != 33; then
 	echo "FAIL: $count error messages, expected 33."
 	$GREP '^E ' $BASE/log/error_log
